src/Connector.js

```diff
@@ -392,15 +394,22 @@ class Connector {
     this.onReconnectedCb = cb;
   }
 
-  // cb(event) where event is { id, sensorId }
+  // Cloud to device (fog)
+
+  // cb(event) where event is { id, [ sensorIds ] }
   async onDataRequested(cb) {
     this.onDataRequestedCb = cb;
   }
 
-  // cb(event) where event is { id, [sensorIds] }
+  // cb(event) where event is { id, [{ sensorId, value }] }
   async onDataUpdated(cb) {
     this.onDataUpdatedCb = cb;
   }
+
+  // cb(event) where event is { id }
+  async onDeviceUnregistered(cb) {
+    this.onDeviceUnregistered = cb;
```

Comment:
Oops, missed the `Cb` here! :sweat_smile: 

```suggestion
    this.onDeviceUnregisteredCb = cb;
```